src/main.cpp

```diff
@@ -8,7 +8,7 @@ int main(int argc, const char* argv[])
     auto args = regexbench::parse_options(argc, argv);
     if (args.quiet)
       return regexbench::exec(args);
-    return regexbench::exec(args, regexbench::realtimeReport);
+    return regexbench::exec(args, regexbench::realtimeReport, nullptr);
```

Comment:
We may omit the last parameter `nullptr` (To reduce the number of changed files^^)